src/main/java/com/dougnoel/sentinel/steps/TableSteps.java

```diff
@@ -145,4 +145,11 @@ public static void storeTableValues(int pageNumber, String tableName) {
     public static void clickColumnHeaderToSort(String columnName, String tableName) {
     	getElementAsTable(tableName).clickColumnHeader(columnName);
 	}
-}
+
+	@When("^I find the (\\d+)(?:st|nd|rd|th) column in the (.*?) and click the cell in the (\\d+)(?:st|nd|rd|th) row$")
+	public static void clickCoordsInTable(int xCoord, String tableName, int yCoord) {
+		--yCoord;
```

Comment:
maybe im being pedantic but you could just do:
getElementAsTable(tableName).clickElementAtCoord(xCoord,--yCoord);

Reply:
Done